compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/lower/BridgeLowering.kt

```diff
@@ -74,7 +75,7 @@ private class BridgeLowering(val context: JvmBackendContext) : ClassLoweringPass
         if (irFunction.isMethodOfAny()) return
 
         if (irFunction.origin === IrDeclarationOrigin.FAKE_OVERRIDE &&
-            irFunction.overriddenSymbols.all { it.owner.modality !== Modality.ABSTRACT && !it.owner.comesFromJava() }
+            irFunction.overriddenSymbols.all { (if ((it.owner.parent as? IrClass)?.isInterface == true) it.owner.hasJvmDefault() else it.owner.modality !== Modality.ABSTRACT) && !it.owner.comesFromJava() }
```

Comment:
`!it.owner.comesFromJava() ` looks wrong: java also generated bridges in interfaces and classes

Reply:
@max-kammerer Fixed in latest patch. I didn't include it in previous patch because it breaks 2 bytecodeText tests and I'm still investigating whether it is the IR backend or tests to blame:

`abstract class A8 : java.util.ArrayList<String>()` had a few bridges before this patch. Now it has no bridges. Although it looks unnecessary to me, the old backend (and the IR backend before this patch) expects the bridges. Are they necessary?

I've disabled those 2 tests for JVM_IR.

Reply:
Oh, yes, it's necessery: it's a bridge beetwen Kotlin  and Java collections - so it should be more wise filtering for java declarations here or later with restored '!it.owner.comesFromJava()' . //cc @gogabr

Reply:
@max-kammerer Done. Please take a look again.